components/table/Table.tsx

```diff
@@ -269,7 +269,8 @@ function InternalTable<RecordType extends object = any>(
   /**
    * Controlled state in `columns` is not a good idea that makes too many code (1000+ line?) to read
    * state out and then put it back to title render. Move these code into `hooks` but still too
-   * complex. We should provides Table props like `sorter` & `filter` to handle control in next big version.
+   * complex. We should provides Table props like `sorter` & `filter` to handle control in next big
+   * version.
```

Comment:
Don't update locale and table

Reply:
Resolved. Probably because of a git commit hook. I use 'git --no-verify' resolved it.